### ISSUE TITLE
Per capita figure for Census dollars

### ISSUE THREAD

**Issue by avickers** [cityofaustin/census2020#34]
tl;dr: Can we summarize the amount of Federal dollars that will be steered on a per capita basis?

---
Huge numbers are hard to comprehend, and it's hard for individuals to understand how one person, especially from these HTC populations, can contribute to or benefit from them.

I also think the numbers will be less controversial in this format.  Variance of a hundred billion dollars is something that people will argue fiercely over.  If you divide that by 320 million, then the difference becomes $312.  Not something that is likely to set passions alight.

When we tell people why they should complete the Census, we want to give them a number that their individual contribution will make, that it's in a range that they can both relate to and believe, and that will be compelling enough to make them be willing to fill out the form.

**Comment by mateoclarke** [comment]
From talking with John, he said:

> - County Tax Collector says $1,500 per person
> - GW study concludes that its hard to get a strict number on a county level because it varies for rural vs urban hard to settle on one number. 

Tara is going to add more info on the figures they want to use on the site.

**Comment by tara-olson-atx** [comment]
[2020_Census_Complete_Count_Committee_One-Pager.pdf](https://github.com/cityofaustin/census2020/files/3888415/2020_Census_Complete_Count_Committee_One-Pager.pdf)

Attached is a one-pager that the Austin-Travis County Census Action Team created for the Complete Count Committee members. Our Census Action Team is made up of dedicated volunteers from Travis 
County, The City of Austin, The U.S. Census Bureau Austin Regional Office, Center for Public Policy Priorities, and NALEO. 

One of the figures I like to use that is easy to understand is: 
Every 10 years, the U.S. Census
Bureau oversees a national
survey to determine population
totals and breakdowns by sex,
age, race and other factors that
guide the allocation of more
than $800 billion in federal
funding to programs across
the U.S. If Texas residents are
undercounted by even 1 percent,
the state could lose at least
$300 million per year.
